src/libraries/System.IO/tests/Stream/Stream.NullTests.cs

```diff
@@ -172,6 +189,22 @@ public static async Task TestNullTextReaderAsync(TextReader input)
             input.Dispose();
         }
 
+        [Theory]
+        [MemberData(nameof(NullReaders))]
+        public static async Task TestNullTextReaderAsync_DisposeAsync(TextReader input)
+        {
+            var chars = new char[2];
+            Assert.Equal(0, await input.ReadAsync(chars, 0, chars.Length));
+            Assert.Equal(0, await input.ReadAsync(chars.AsMemory(), default));
+            Assert.Equal(0, await input.ReadBlockAsync(chars, 0, chars.Length));
+            Assert.Equal(0, await input.ReadBlockAsync(chars.AsMemory(), default));
+            Assert.Null(await input.ReadLineAsync());
+            Assert.Null(await input.ReadLineAsync(default));
+            Assert.Equal("", await input.ReadToEndAsync());
+            Assert.Equal("", await input.ReadToEndAsync(default));
+            await input.DisposeAsync();
```

Comment:
I wanted to avoid breaking chagnes, since Dispose is calling "Close()" on the underlying stream and DisposeAsync() is calling "DisposeAsync()" on the underlying stream. But if this isn't a concern I will happily change this.

Reply:
@Jozkee what do you think we should do here?

Reply:
I still think you should apply my suggestion here because `TestNullTextReaderAsync` above is already testing Async conterparts.